### PR TITLE
Migrate to Clingo 5 and Python 3

### PR DESCRIPTION
**Pull request by pauleve**
Tests and manual validation with examples for caspo control work as expected.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-seaborn==0.7.0
+joblib
+networkx
+numpy
+pandas
+pydot
+pyparsing
+scikit-learn
+scipy
+seaborn
```
Comment (svidela) [inline requirements.txt:10]: I'd pin specific (latest if possible) versions in both requirements.txt and meta.yaml before making the next release

Comment (pauleve) [reply requirements.txt:10]: Ha, I'm quite against this :wink:  I think it creates much more troubles than it solves whenever other packages are installed alongside. I prefer having early bug reports due to API breakage, and (validated) Docker images really help on this; I also plan to include caspo in the CoLoMoTo Docker ecosystem https://colomoto.github.io/colomoto-docker where we work for providing reproducible software environments.

**Comment by pauleve** [comment]
@svidela any idea to make codacy happy with import clingo?

**Comment by svidela** [comment]
> @svidela any idea to make codacy happy with import clingo?

I don't have experience with codacy, but it seems to use pylint. In that case, it should be possible to just ignore it using:

```
import clingo  # pylint: disable=import-error
```

**Pull request merged**